spec/javascripts/support/jasmine_helper.rb

```diff
@@ -1,7 +0,0 @@
-require "jasmine_selenium_runner/configure_jasmine"
-
-class ChromeHeadlessJasmineConfigurer < JasmineSeleniumRunner::ConfigureJasmine
-  def selenium_options
-    { options: GovukTest.headless_chrome_selenium_options }
```

Comment:
Just to bear in mind for when this progresses more, without this we lose a useful env var flag so that the tests can run within a Docker environment: https://github.com/alphagov/govuk_test/blob/53a8c728caecf412b48ff807437d0c75b5992f13/lib/govuk_test.rb#L30 we may have to think about whether there's a nice way we can share this across apps.